include/deal.II/distributed/tria_base.h

```diff
@@ -246,6 +246,24 @@ namespace parallel
   template <int dim, int spacedim = dim>
   using Triangulation DEAL_II_DEPRECATED = TriangulationBase<dim, spacedim>;
 
+  template <int dim, int spacedim = dim>
+  class DistributedTriangulationBase
+    : public dealii::parallel::TriangulationBase<dim, spacedim>
+  {
+  public:
+    /**
+     * Constructor.
+     */
+    DistributedTriangulationBase(
+      MPI_Comm mpi_communicator,
+      const typename dealii::Triangulation<dim, spacedim>::MeshSmoothing
+                 smooth_grid = (dealii::Triangulation<dim, spacedim>::none),
+      const bool check_for_distorted_cells = false);
+
+    virtual bool
+    do_construct_multigrid_hierarchy() const = 0;
```

Comment:
this function needs documentation. I actually have no idea what it does from the name. "do_...." means that it does something?

Reply:
@tjhei @bangerth I have renamed the method to `is_multilevel_hierarchy_constructed` as propsed by @tjhei  and have added some documentation.